meteor/client/lib/ModalDialog.tsx

```diff
@@ -295,16 +302,27 @@ class ModalDialogGlobalContainer0 extends React.Component<
 			queue: [],
 		}
 	}
+
 	public addQueue(q: ModalDialogQueueItem) {
 		const queue = this.state.queue
 		queue.push(q)
 		this.setState({
 			queue,
 		})
 	}
+
 	public queueHasItems(): boolean {
 		return this.state.queue.length > 0
 	}
+
+	public removeAllQueueItems(): void {
+		const queue = this.state.queue
+		queue.splice(0)
+		this.setState({
+			queue,
+		})
```

Comment:
Minor thing. Since we only set a single attribute, then having it on a single line is fine.
```suggestion
		this.setState({ queue })
```